modules/org.restlet/src/main/java/org/restlet/engine/header/HeaderUtils.java

```diff
@@ -237,8 +237,8 @@ public static void addEntityHeaders(Representation entity,
                     && ALGORITHM_MD5.equals(entity.getDigest().getAlgorithm())) {
                 addHeader(
                         HEADER_CONTENT_MD5,
-                        new String(org.restlet.engine.util.Base64.encode(entity
-                                .getDigest().getValue(), false)), headers);
+                        new String(java.util.Base64.getEncoder().encode(entity
```

Comment:
Here again ?

Reply:
@quilicicf not at all, this is because of code generation.
The GWT edition does not support the java.util.Base64 class because of limitations of the JDK used for GWT.
It is very difficult to generate the right source code for the GWT edition, if the "import" section contains unsupported packages.